Makefile

```diff
@@ -154,8 +161,22 @@ KUSTOMIZE = $(PROJECT_DIR)/bin/kustomize
 kustomize:
 	@GOBIN=$(PROJECT_DIR)/bin go install sigs.k8s.io/kustomize/kustomize/v4@v4.5.7
 
+.PHONY: scheduler-plugins
+scheduler-plugins:
+	GOPATH=/tmp go get -d sigs.k8s.io/scheduler-plugins@$(SCHEDULER_PLUGINS_VERSION)
+
 .PHONY: scheduler-plugins-crd
-scheduler-plugins-crd:
-	GOPATH=/tmp go get sigs.k8s.io/scheduler-plugins@$(SCHEDULER_PLUGINS_VERSION)
+scheduler-plugins-crd: scheduler-plugins
 	mkdir -p $(PROJECT_DIR)/dep-crds/scheduler-plugins/
 	cp -f /tmp/pkg/mod/sigs.k8s.io/scheduler-plugins@$(SCHEDULER_PLUGINS_VERSION)/manifests/coscheduling/* $(PROJECT_DIR)/dep-crds/scheduler-plugins
+
+.PHONY: scheduler-plugins-chart
+scheduler-plugins-chart: scheduler-plugins-crd
+	mkdir -p $(PROJECT_DIR)/dep-manifests/scheduler-plugins/
+	cp -rf /tmp/pkg/mod/sigs.k8s.io/scheduler-plugins@$(SCHEDULER_PLUGINS_VERSION)/manifests/install/charts/as-a-second-scheduler/* $(PROJECT_DIR)/dep-manifests/scheduler-plugins
+	mkdir -p $(PROJECT_DIR)/dep-manifests/scheduler-plugins/crds
+	cp -f /tmp/pkg/mod/sigs.k8s.io/scheduler-plugins@$(SCHEDULER_PLUGINS_VERSION)/manifests/appgroup/crd.yaml $(PROJECT_DIR)/dep-manifests/scheduler-plugins/crds/appgroup.diktyo.x-k8s.io_appgroups.yaml
+	cp -f /tmp/pkg/mod/sigs.k8s.io/scheduler-plugins@$(SCHEDULER_PLUGINS_VERSION)/manifests/networktopology/crd.yaml $(PROJECT_DIR)/dep-manifests/scheduler-plugins/crds/networktopology.diktyo.x-k8s.io_networktopologies.yaml
```

Comment:
Yes, we must install all CRDs. If not installing, the scheduler-plugins-scheduler crashes since unavailable to watch custom resources.
 
> E0331 18:35:23.310663       1 reflector.go:140] sigs.k8s.io/scheduler-plugins/pkg/generated/informers/externalversions/factory.go:117: Failed to watch *v1alpha1.ElasticQuota: failed to list *v1alpha1.ElasticQuota: the server could not find the requested resource (get elasticquotas.scheduling.x-k8s.io)

Reply:
That's sad. Ideally people should be able to use some plugins without having to install all CRDs.

Do you mind opening an issue for this and linking it here?

Reply:
> Do you mind opening an issue for this and linking it here?

Sure. Maybe, ideally, we want to restrict watching CR by specifying GVR in flags.